source/cpp/source/DefaultCommandHandler.cpp

```diff
@@ -456,6 +456,33 @@ Response getComboBoxNumItems (const Command & command)
     return Response::fail (error);
 }
 
+Response getComboBoxItems (const Command & command)
+{
+    auto comboBoxVariant = getComponent<juce::ComboBox> (command);
+    if (std::holds_alternative<juce::ComboBox *> (comboBoxVariant))
+    {
+        auto * comboBox = std::get<juce::ComboBox *> (comboBoxVariant);
+        const auto numItems = comboBox->getNumItems ();
+
+        if (numItems == 0)
+        {
+            return Response::fail (juce::String ("ComboBox is empty"));
+        }
+
+        juce::StringArray items;
+
+        for (auto i = 0; i < numItems; ++i)
+        {
+            items.add (comboBox->getItemText (i));
+        }
+
+        return Response::ok ().withParameter ("items", items);
```

Comment:
I don't think this should fail if there are no items - it can just return an empty array. Should be able to just remove the check for `== 0`:

```suggestion
        juce::StringArray items;

        for (auto i = 0; i < numItems; ++i)
        {
            items.add (comboBox->getItemText (i));
        }

        return Response::ok ().withParameter ("items", items);
```

Reply:
Sure thing! I committed your suggested change after locally validating it.